library/Requests/Transport/cURL.php

```diff
@@ -130,6 +130,25 @@ public function __destruct() {
 	 * @return string Raw HTTP result
 	 */
 	public function request($url, $headers = array(), $data = array(), $options = array()) {
+		if (is_array($data) === false && is_string($data) === false) {
```

Comment:
I think this type of notation might be easier to read:
```suggestion
		if (! is_array($data) && ! is_string($data)) {
```
Is there a specific reason to use the `=== false` notation instead?

Reply:
Fixed